entities/OrderedStation.php

```diff
@@ -96,6 +96,16 @@ public function linkedListPathAsArray() {
     return array_map('intval', $r);
   }
 
+  /** Will take a coordinate and return the closest stationid
+   *
+   * Will take a coordinate and return the closest stationid
+   *
+   * @param float $coord Coordinates from client
+   * @return [OrderedStation] [NYI]
+   */
+  public function getStationFromCoord($coord){
```

Comment:
It should be a 2 element array containing the two floats, oversight on my part. That being said I was thinking that the two floats would be passed into this query we're creating to get the closest station linestring in postGIS